signup.php

```diff
@@ -1,9 +1,13 @@
 <html lang="en">
 <head>
+<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
 	<title>Sign Up Page</title>
+	<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl" crossorigin="anonymous">
+	<link rel="stylesheet" type="text/css" href="css/main.css">
 </head>
-<body>
-Please sign up
+<body class="signupBody">
```

Comment:
why would signup have a completely different body class? shouldn't there be a container? also theres no nav bar here...

Reply:
this page should follow the same form as all the other pages.

Reply:
> why would signup have a completely different body class? shouldn't there be a container? also theres no nav bar here...

its so I can have the background image for this specific page, not sure if this is best practice

Reply:
deff not... i didnt even look at the result but it should always follow the same formatting. if you want a background image for the page, make a container that wraps this pages content.